src/vscode-dts/vscode.proposed.envCollectionOptions.d.ts

```diff
@@ -0,0 +1,56 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+declare module 'vscode' {
+
+	// https://github.com/microsoft/vscode/issues/179476
+
+	/**
+	 * Options applied to the mutator.
+	 */
+	export interface EnvironmentVariableMutatorOptions {
+		/**
+		 * Apply to the environment just before the process is created.
+		 *
+		 * Defaults to true.
+		 */
+		applyAtProcessCreation?: boolean;
```

Comment:
It's not following: https://github.com/Microsoft/vscode/wiki/Extension-API-guidelines#undefined-is-false

Reply:
Let's bring this up at the API sync, undefined not being true would be harmful here imo.